Maple2.Server.Game/Manager/GuildManager.cs

```diff
@@ -48,6 +49,10 @@ public class GuildManager : IDisposable {
         if (Guild != null) {
             foreach (GuildMember member in Guild.Members.Values) {
                 member.Dispose();
+                if (member.CharacterId == session.CharacterId) {
+                    using GameStorage.Request db = session.GameStorage.Context();
+                    db.SaveGuildMember(member);
```

Comment:
same as comment on `ClubManager`, `WorldServer` should be source of truth and no saves should be needed from `GameServer`.